src/token/token.ts

```diff
@@ -281,10 +282,10 @@ export class CollationChangeToken extends Token {
   declare event: 'sqlCollationChange';
 
   type: 'SQL_COLLATION';
-  oldValue: Buffer;
-  newValue: Buffer;
+  oldValue: Collation | undefined;
+  newValue: Collation | undefined;
```

Comment:
Would a collation envchange ever be sent without a new collation?

Reply:
Practically, probably not, but technically the server can send back a zero length `BVARBYTE` value.